ridgepole-rails.gemspec

```diff
@@ -20,6 +20,6 @@ Gem::Specification.new do |s|
   s.files = Dir["lib/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", ">= 5.0.0"
-  s.add_dependency "ridgepole", "~> 0.6.5"
+  s.add_dependency "ridgepole", "~> 0.7.0"
```

Comment:
Do you want to continue supporting `0.6.5`? If so, this should be `">= 0.6.5", "< 0.8.0"`. I wrote [Dependabot](https://dependabot.com) to make these kind of updates automatically whenever a new version of a sub-dependency gets released, if you want to take a look :octocat: